WkHtmlToXSharp.Linux32/WkHtmlToXSharp.Linux32.csproj

```diff
@@ -64,6 +70,7 @@
     <None Include="WkHtmlToXSharp.Linux32.nuspec">
       <SubType>Designer</SubType>
     </None>
+    <None Include="WkHtmlToXSharp.snk" />

```

Comment:
May you update the pull-req so all projects reference the same .snk file (actually the one included within WkhtmlToXSharp folder?)